functions/index.js

```diff
@@ -6,3 +6,12 @@ const functions = require('firebase-functions');
 // exports.helloWorld = functions.https.onRequest((request, response) => {
 //  response.send("Hello from Firebase!");
 // });
+
+exports.triggerTest = functions.firestore.document('stock/{stockId}').onCreate(event => {
+    const newValue = event.data.data();
+    console.log(event);
+    if(newValue.amount > 10) {
+      console.log('a lot of stock');
+    }
+    return true;
+});
```

Comment:
our firebase functions will live inside our `functions/index` file, this one is good but please append the function with notes on its functionality

Reply:
kk) this is just a sample :)